src/wiki/windows-insiders.md

```diff
@@ -129,31 +129,31 @@ If you don't want to participate in the Windows Insider Program anymore, you can
 
 1. Open *Settings*, click on *Updates & Security*
 
-![](./img/windows-insider/11/settings-home.png)
+![Settings](./img/windows-insider/11/settings-home.png)
 
 2. Click on **Windows Insider Program**
 
-![](./img/windows-insider/11/settings-select.png)
+![Insider Program](./img/windows-insider/11/settings-select.png)
 
 3. Click on **Stop getting preview builds**.
 
-![](./img/windows-insider/11/leaving-get-started.png)
+![Leaving](./img/windows-insider/11/leaving-get-started.png)
 
 4. Click on **Leaving the Insider Program**.
 
-![](./img/windows-insider/11/unenroll-immediately.png)
+![Unenroll](./img/windows-insider/11/unenroll-immediately.png)
 
 5. You will be redirected to an official website. Click on *Leave the Program now*.
 
-![](./img/windows-insider/10/leaving-the-program.png)
+![Really leaving](./img/windows-insider/10/leaving-the-program.png)
```

Comment:
Please use more descriptive alts. "Really leaving" isn't really descriptive for users that rely on screenreaders.